docs/debugging.md

```diff
@@ -56,6 +61,21 @@ datasets:
 >[!Tip]
 > If you prefer to watch a video, rather than read, you can skip to the [video tutorial](#video-tutorial) below (but doing both is recommended).
 
+### Setup
+
+Make sure you have an editable install of Axolotl.  If you don't, run the following from the root of this project:
```

Comment:
maybe explain that an editable install means they can update the code for live changes without needing to reinstall